app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def superadmin_or_department_manager?
     superadmin? || department_manager?
   end
 
+  def department?
```

Comment:
I did name it longer first, and the linter suggested me to shorten its name. But yeah, I'm willing to change the name if people prefer it to be changed.